src/Client.php

```diff
@@ -226,7 +228,13 @@ public function getCookieJar()
         return new CookieJar($this->webDriver);
     }
 
-    public function waitFor(string $cssSelector, int $timeoutInSecond = 30, int $intervalInMillisecond = 250): object
+    /**
+     * @throws NoSuchElementException
+     * @throws TimeOutException
+     *
+     * @return object
```

Comment:
shouldn't this say it returns a WebDriverElement instead ? `->until` is documented as returning the return value of the callable. If you just rely on that, the return type would be `mixed`, not `object`. And if you take the callable into account, you can be more precise than `object` (which is not much better than `mixed` as far as autocompletion is concerned)